src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -298,7 +298,11 @@ protected function get_current_rest_url( array $args = array() ) {
 		$keys    = array_intersect_key( $flipped, $values );
 
 		if ( ! empty( $keys ) ) {
-			$url = add_query_arg( array_combine( array_values( $keys ), array_values( $values ) ), $url );
+			$param = array_fill_keys( array_values( $keys ), '' );
```

Comment:
Nitpicking here: for the sake of clarity could you rename this from `$param` to `$paremeters`?